ui/app/appointments/controllers/manage/calendar/appointmentsDayCalendarController.js

```diff
@@ -54,14 +54,21 @@ angular.module('bahmni.appointments')
                     uiCalendarConfig.calendars[calendar].fullCalendar('changeView', view);
                 };
 
-                var isUserPrivilegedToCreateAppointment = function () {
+                function isOwnAppointmentUserPrivilegedToCreateAppointment (privilege, provider) {
```

Comment:
I think this "isOwnAppointmentUserPrivilegedToCreateAppointment" is only used from "isUserPrivilegedToCreateAppointment()" - check the comment against the "isUserPrivilegedToCreateAppointment" method.